### DISCUSSION TITLE
Need help with orbot please

### DISCUSSION THREAD

**Discussion by gofukrself** [guardianproject/orbot-android#711] [General]
Hi, for the last few weeks orbot is not working, it just keeps saying it's starting. I have the latest update. I have tried uninstalling the updates and then reinstall, I've tried deleting the app and reinstall and all have same result......nothing. I sent screenshots umm not sure if they will come through or not. I have also gotten a low memory warning as to why it won't engage. I have 15g free on my SD card and 2or3g free on system. I have android 10 I am using a Lenovo TB-8505F and android is updated to latest version. 

![Screenshot_2022-07-13-09-41-33-872](https://user-images.githubusercontent.com/103591120/178776802-02c64be5-e6fc-4b38-8c5a-d41919c6e292.jpeg)
![Screenshot_2022-07-13-09-41-51-836](https://user-images.githubusercontent.com/103591120/178776804-20201170-2791-4fff-b9f8-7118e3a17133.jpeg)
![Screenshot_2022-07-13-09-42-56-583](https://user-images.githubusercontent.com/103591120/178776806-51f18ac0-3fad-4ce1-b6c0-66bed173bb25.jpeg)
![Screenshot_2022-07-13-09-43-32-042](https://user-images.githubusercontent.com/103591120/178776810-8382da40-e518-4e84-944d-3a5198e33b7d.jpeg)
![Screenshot_2022-07-13-09-43-46-016](https://user-images.githubusercontent.com/103591120/178776818-6a416ef4-28b7-47d6-bfbf-b7e1cffffd33.jpeg)
![Screenshot_2022-07-13-09-44-07-986](https://user-images.githubusercontent.com/103591120/178776822-cad8d0b4-bce1-412b-a9f0-10fdfe405127.jpeg)
![Screenshot_2022-07-13-09-44-31-103](https://user-images.githubusercontent.com/103591120/178776824-c0765644-a3c2-4275-947a-233b1a231eae.jpeg)
![Screenshot_2022-07-13-09-47-56-782](https://user-images.githubusercontent.com/103591120/178776826-c45f1b57-e347-4c4f-bdd7-aea424ad68fa.jpeg)

**Comment by bitmold** [comment]
Hey @gofukrself, we are working on a new release right now !

**Reply by gofukrself** [reply]
Did you just tell me to go fuck myself?

**Reply by gofukrself** [reply]
Lmfao just kidding. The way the message is worded is perfect  lmfao
